report_qweb_txt/models/report.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+
+class Report(models.Model):
+    _inherit = "report"
+
+    @api.model
+    def _get_report_from_name(self, report_name):
```

Comment:
I don't see the advantage of calling super... it seems like a waste a resources for me.
I decided to raise an error if nothing is found in order to have a clean error msg instead of having a crash later on in the code (because I guess it's what happens if nothing is found).
I think the only clean solution is to propose a PR to fix the original code...